python/lsst/afw/image/utils.py

```diff
@@ -69,29 +68,6 @@ def defineFilter(name, lambdaEff, lambdaMin=np.nan, lambdaMax=np.nan, alias=[],
             Filter.defineAlias(name, a)
 
 
-def defineFiltersFromPolicy(filterPolicy, reset=False):
-    """Process a Policy and define the filters"""
-
-    if reset:
-        Filter.reset()
-        FilterProperty.reset()
-    #
-    # Process the Policy and define the filters
-    #
-    policyFile = pexPolicy.DefaultPolicyFile(
-        "afw", "FilterDictionary.paf", "policy")
-    defPolicy = pexPolicy.Policy.createPolicy(
-        policyFile, policyFile.getRepositoryPath(), True)
-
-    filterPolicy.mergeDefaults(defPolicy.getDictionary())
-
-    for p in filterPolicy.getArray("Filter"):
-        Filter.define(FilterProperty(p.get("name"), p))
-        if p.exists("alias"):
-            for a in p.getArray("alias"):
-                Filter.defineAlias(p.get("name"), a)
-
-
```

Comment:
There are three uses of this in the **lsst/ctrl_prepdc3b** pakcage (whatever that is!):
https://github.com/lsst/ctrl_prepdc3b/blob/master/python/splitCfhtForDc3b2.py#L48
https://github.com/lsst/ctrl_prepdc3b/blob/master/python/splitCfhtForDc3b3.py#L55
https://github.com/lsst/ctrl_prepdc3b/blob/master/python/splitCfhtForDc3b4.py#L31

Reply:
That package is definitely dead. I'm surprised we haven't moved it to `lsst-dm/legacy_` (I'm following up on this on slack).